### ISSUE TITLE
Windows Install Instruction are incorrect

### ISSUE THREAD

**Issue by jlong29** [simbody/simbody#495]
The recommended version for installing Simbody using Visual Studio 2015 should be 3.5.3 and not 3.5.1. 3.5.1 does not compile under Visual Studio 2015, though it does under Visual Studio 2013.

**Comment by chrisdembia** [comment]
Thanks for reporting. Are you talking about this specific part:

![image](https://cloud.githubusercontent.com/assets/846001/15443145/cdfec23e-1e9a-11e6-8ee6-6565b7faaac8.png)

?

**Comment by jlong29** [comment]
My pleasure. That's the line. The second one should be "$git checkout
SImbody-3.5.3"

On Fri, May 20, 2016 at 5:55 PM, Christopher Dembia <
notifications@github.com> wrote:

> Thanks for reporting. Are you talking about this specific part:
>
> [image: image]
> <https://cloud.githubusercontent.com/assets/846001/15443145/cdfec23e-1e9a-11e6-8ee6-6565b7faaac8.png>
>
> ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/simbody/simbody/issues/495#issuecomment-220727804>
>



-- 

John D. Long II, PhD
Post-doctoral fellow
Neuroscience Institute
New York University, Langone Medical Center
Tel: 212-263-9119; Fax: 212-263-9170
http://www.med.nyu.edu/buzsakilab/

**Comment by chrisdembia** [comment]
Would you be interested in submitting a PR to update the README? You can either just change `3.5.1` to `3.5.3` (there are multiple occurrences), or remove that line and the explanation of it, or give an explanation in the code block

```
$ git clone https://github.com/simbody/simbody.git C:/Simbody-source
$ git checkout Simbody-3.5.1 # For example, if you want a specific release. 
```

**Comment by Gjacquenot** [comment]
Hi @jlong29 , @chrisdembia , I have created a PR #496, that updates this version number.

**Comment by sherm1** [comment]
Resolved by #496.
